crates/gitbutler-branch-actions/src/branch.rs

```diff
@@ -21,6 +23,33 @@ use std::{
     vec,
 };
 
+pub(crate) fn get_uncommited_files(
+    context: &CommandContext,
+    _permission: &WorktreeReadPermission,
+) -> Result<Vec<RemoteBranchFile>> {
+    let repository = context.repository();
+    let head_commit = repository
+        .head()
+        .context("Failed to get head")?
+        .peel_to_commit()
+        .context("Failed to get head commit")?;
+
+    let files = workdir(repository, &head_commit.id())
```

Comment:
I think this really should remain as `gitbutler_diff::workdir()` so it's clear(er) that it's a diff.